analysis/stripe/analysis.sql

```diff
@@ -0,0 +1,87 @@
+with
+
+invoices as (
+  select *
+  from {{env.schema}}.stripe_invoices_transformed
+),
+
+plan_changes as (
+  select
+    invoices.customer as customer,
+    date_trunc('month', invoices.date) as month,
+    invoices.total as now,
+    prior_month_invoices.total as before,
+    invoices.total - prior_month_invoices.total as change
+  from
+    invoices
+    left outer join
+      invoices prior_month_invoices
+    on
+      date_trunc('month', add_months(invoices.date, -1)) = date_trunc('month', prior_month_invoices.date)
+      and invoices.customer = prior_month_invoices.customer
+  where
+    invoices.forgiven is not true
+    and invoices.paid is true
```

Comment:
@jthandy Also, the join doesn't seem to work in case of irregular payments (say, someone getting an invoice on March 3 and another one on March 30). Also, what happens if an invoice is missing for one month?